src/main/java/org/myrobotlab/service/Ads1115.java

```diff
@@ -438,12 +438,12 @@ public void attach(I2CController controller, String deviceBus, String deviceAddr
   }
 
   @Override
-  public void attach(PinArrayListener listener) {
+  public void attachPinArrayListener(PinArrayListener listener) {

```

Comment:
this method should be moved to the interface as a default method and removed from all services.